cmd/orb-cli/common/common.go

```diff
@@ -493,6 +520,10 @@ func (s *Signer) Headers() jws.Headers {
 		headers[jws.HeaderAlgorithm] = "EdDSA"
 	}
 
+	if s.publicKey.Crv == "P-256" || s.publicKey.Crv == "P-384" || s.publicKey.Crv == "P-521" {
+		headers[jws.HeaderAlgorithm] = "ES256"
```

Comment:
this is not correct for P-284/P-521. see https://www.rfc-editor.org/rfc/rfc7518#section-3.1

create a map for the algs like  you did above with the curve